src/main/java/com/akademia/projectplanner/dto/TaskDto.java

```diff
@@ -1,18 +1,24 @@
 package com.akademia.projectplanner.dto;
 
+import com.akademia.projectplanner.entity.UserEntity;
 import lombok.Getter;
 import lombok.Setter;
 
 import javax.validation.constraints.NotEmpty;
+import java.time.LocalDate;
 
 @Getter
 @Setter
 public class TaskDto {
 
+  private Long id;
+
   @NotEmpty(message = "Task name cannot be empty!")
   private String name;
 
   private String status;
   private String description;
   private String deadline;
+  private LocalDate startDate;
+  private UserEntity userEntity;
```

Comment:
Obiekty dto czyli służące do transferu danych do frontu są dodatkową warstwą "zamykającą" kod. Nie chcemy aby wrażliwe dane były przesyłane niepotrzebnie przez sieć, dlatego nie powinniśmy w dto zawierać całej encji użytkownika (druga sprawa, że jeśli enjcja ma wile pól to zapychamy też sieć). Wiem, że DTO i mappera do User'a ma zrobić Agnieszka więc jej zadaniem będzie to zmienić ale piszę na przyszłość  :)